errors/Readme.md

```diff
@@ -659,7 +195,7 @@ Note: Studio has two workflows:
 Electron app: Credentials from the UI → Introspection → Prisma schema → Valid Prisma project
 Web app: `prisma2 dev` → Provides Prisma schema i.e a Valid Prisma project
 
-Since studio uses Photon for query building. It relays the same error messages as Photon. Additionally, it relays the following errors from the SDK: `P3000`, `P5000`
+Since studio uses Photon for query building. It relays the same error messages as Photon. Additionally, it relays the following errors from the SDK: `P3000`.
```

Comment:
Why where the additional errors here wrong?

Reply:
Those codes were schema errors. Since i removed this category i removed the referenced codes as well.